ensemble/ctf/editor.py

```diff
@@ -209,7 +210,11 @@ def _draw_colors(self, color_nodes, gc):
             gc.rect(0, 0, w, h)
             # XXX : We need to pass byte strings instead of unicode strings.
             # See https://github.com/enthought/enable/issues/342
-            gc.linear_gradient(0, 0, w, 0, grad_stops, b'pad', b'userSpaceOnUse')
+            if six.PY2:
+                args = (b'pad', b'userSpaceOnUse')
+            else:
+                args = ('pad', 'userSpaceOnUse')
```

Comment:
Since this is in the rendering code path _and_ it never changes, can you move it out to a module-level constant?

Reply:
corrected.

Reply:
Where?

Reply:
sorry. i corrected it locally. haven't pushed yet.